### PR TITLE
supervisor-upgrade: restructure docs to provide upgrade instructions

### PR DESCRIPTION
**Pull request by xginn8**
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

**Comment by xginn8** [comment]
Screenshot_2021-01-12 Self-service supervisor upgrades - Balena Documentation

for reference

File: pages/reference/supervisor/supervisor-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Updates can also be issued to multiple devices in the same application with the same CPU architecture. From the device
+list, click the checkbox to the left of any online devices you wish to update. Then click the *Actions* button in the
+upper-right corner of the dashboard, followed by *Update supervisor*:
+
+![Supervisor batch upgrade](/img/common/updates/supervisor-batch-upgrade.svg)
+
+Additionally, these devices can be scheduled for an offline device. The update will be performed once the device comes
+back online and successfully connects to the {{ $names.cloud.lower }} backend.
+
+__Note:__ Only devices running {{ $names.os.lower }} v2.12 or greater are able to upgrade the Supervisor independently.
```
Comment (xginn8) [inline pages/reference/supervisor/supervisor-upgrades.md:27]: this version is totally arbitrary and pending a product call, but including it as a placeholder.

File: pages/reference/supervisor/supervisor-upgrades.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+For more information on specific limitations when upgrading the balena Supervisor, see the [upgrade policy](/reference/supervisor/upgrade-policy).
```
Comment (vipulgupta2048) [inline pages/reference/supervisor/supervisor-upgrades.md:29]: If you can convert this link into a footer link for consistency's sake. 
```
For more information on specific limitations when upgrading the balena Supervisor, see the [upgrade policy][upgrade-policy]

[upgrade-policy]:/reference/supervisor/upgrade-policy
```

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
@xginn8 Looking good :+1:  
Regarding the screenshot of the dashboard, usually, people get confused with the stripes that are present on staging. Since the feature has hit production it would really help if you can click the screenshot again and now without stripes for consistency.

You can also remove stripes in your staging dashboard header by navigating to preferences > experimental options.

File: pages/reference/supervisor/supervisor-upgrades.md (hunk 2 above)
Comment (xginn8) [reply pages/reference/supervisor/supervisor-upgrades.md:29]: there's not really a standard within the docs at the moment. there are lots of both types of links. i'm fine converting this, but if we value consistency we should enforce it across the board.

File: pages/reference/supervisor/supervisor-upgrades.md (hunk 1 above)
Comment (xginn8) [reply pages/reference/supervisor/supervisor-upgrades.md:27]: MC has been determined to be an appropriate cutoff, see https://www.flowdock.com/app/rulemotion/r-product/threads/NLB1gISigptXnkF9fNmNWPVQrsR

**Comment by xginn8** [comment]
@balena-ci I self-certify!

**Comment by xginn8** [comment]
@vipulgupta2048 please don't `request changes` on drafts, especially for things that can be fixed trivially since it requires you to take a second look and remove your block.

**Comment by xginn8** [comment]
@vipulgupta2048 can you please remove your block on this PR, so it can be merged?

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**